pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1272,4 +1274,28 @@ public void setPreferLaterVersions(boolean preferLaterVersions) {
     public int getWebSocketConnectionsPerBroker() { return webSocketConnectionsPerBroker; }
 
     public void setWebSocketConnectionsPerBroker(int webSocketConnectionsPerBroker) { this.webSocketConnectionsPerBroker = webSocketConnectionsPerBroker; }
+
+    public static ServiceConfiguration convertFrom(PulsarConfiguration conf) throws RuntimeException {
```

Comment:
Probably we can define this utility under `PulsarConfigurationLoader`?

Reply:
I moved the utility method.